<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conditions</class>
 <widget class="QMainWindow" name="Conditions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Condition:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ifCondition">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="elseCB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Else:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ifelseCondition">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="ifElseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>if-else</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="conditionalButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Conditional</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Condition</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conditionalCond">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Action 1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conditionalElse">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch</string>
    </property>
   </widget>
   <widget class="QListWidget" name="switchList">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>311</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Case:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="switchCase">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okayButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchAddButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Action 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cond2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Variable Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="switchName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
